.github/workflows/scripts/before_script.sh

```diff
@@ -39,7 +39,7 @@ cmd_prefix bash -c "usermod -a -G wheel pulp"
 SCENARIOS=("pulp" "performance" "azure" "gcp" "s3" "stream" "generate-bindings" "lowerbounds")
 if [[ " ${SCENARIOS[*]} " =~ " ${TEST} " ]]; then
   # Many functional tests require these
-  cmd_prefix dnf install -yq lsof which
+  cmd_prefix dnf install -yq dbus-daemon flatpak lsof which
```

Comment:
Should I be concerned about the
```
# WARNING: DO NOT EDIT!
#
# This file was generated by plugin_template, and is managed by it. Please use
# './plugin-template --github pulp_container' to update this file.
#
# For more info visit https://github.com/pulp/plugin_template
```
at the top of this file? Should these additional `dnf install` rather be done somewhere else?

Reply:
Oh, nice catch! Yes, you should create a new file called `.github/workflows/scripts/post_before_script.sh` where you will install the additional packages in the same way. This script is called from `before_script.sh`.

Reply:
done in 914492c284d764a42378cf968b50c93ea6c4a9a9